docs/pages/repo/docs/handbook/dev.mdx

```diff
@@ -27,7 +27,8 @@ You should specify your `dev` task like this in your `turbo.json`.
 {
   "pipeline": {
     "dev": {
-      "cache": false
+      "cache": false,
+      "persistent": true
```

Comment:
We should explain why `persistent: true` is needed in this doc.

Reply:
done